distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso

```diff
@@ -111,38 +133,52 @@ make_fill_default_value_selector column_source=Nothing display:Display=..Always
 
 ## PRIVATE
    Make a filter condition selector.
-make_filter_condition_selector table display:Display=..Always =
-    equatable_types = make_column_ref_by_name_selector table add_text=True add_number=True add_boolean=True add_date=True add_time=True add_date_time=True
-    comparable_types = make_column_ref_by_name_selector table add_text=True add_number=True add_date=True add_time=True add_date_time=True
-    with_text = make_column_ref_by_name_selector table add_text=True
+make_filter_condition_selector table display:Display=..Always cache=Nothing =
+    column_value_type = get_column_value_type table cache
+    column_chooser = make_column_ref_by_name_selector_for_type table display add_nothing=False value_type=column_value_type
+
+    is_all = include_all column_value_type
+    include_text = is_all || column_value_type.is_text
+    include_boolean = is_all || column_value_type.is_boolean
+    include_floating = is_all || column_value_type.is_floating_point
 
     options = Vector.build builder->
-        builder.append (Option "Equals" "..Equal" [["to", equatable_types]])
-        builder.append (Option "Not Equals" "..Not_Equal" [["to", equatable_types]])
-        builder.append (Option "Less Than" "..Less" [["than", comparable_types]])
-        builder.append (Option "Less Than Or Equal" "..Equal_Or_Less" [["than", comparable_types]])
-        builder.append (Option "Greater Than" "..Greater" [["than", comparable_types]])
-        builder.append (Option "Greater Than Or Equal" "..Equal_Or_Greater" [["than", comparable_types]])
-        builder.append (Option "Between" "..Between" [["lower", comparable_types], ["upper", comparable_types]])
-        builder.append (Option "Equals Ignore Case" "..Equal_Ignore_Case" [["to", with_text]])
-        builder.append (Option "Starts With" "..Starts_With" [["prefix", with_text]])
-        builder.append (Option "Ends With" "..Ends_With" [["suffix", with_text]])
-        builder.append (Option "Contains" "..Contains" [["substring", with_text]])
-        builder.append (Option "Like" "..Like" [["pattern", with_text]])
+        builder.append (Option "Equals" "..Equal" [["to", column_chooser]])
+        builder.append (Option "Not Equals" "..Not_Equal" [["to", column_chooser]])
+
+        if column_value_type != Value_Type.Boolean then
```

Comment:
We have a method on `Value_Type` that checks this condition more precisely, would be better to use it:
```suggestion
        if column_value_type.has_ordering then
```

Reply:
Want to exclude Boolean here as well (it has an ordering) but have updated.

Reply:
Ahh, I forgot that Boolean has ordering then. Hmm. Perhaps then `column_value_type.has_ordering && column_value_type != Value_Type.Boolean` to exclude the boolean explicitly?